File: lei_lds2122_grupo2-main/Frontend/src/app/viewModels/responses/UserSearchListVM.ts
import { UserResultVM } from "./UserResultVM";

/**
 * View Model class for the list of users that results from a user search
 */
export class UserSearchListVM {

    /**
     * Construct's an instance of {@link UserSearchListVM}
     * 
     * @param users an array of {@link UserResultVM}
     */
    constructor(
        public users: Array<UserResultVM>
    ){}
}

File: Backend/Data/ViewModels/Response/UserSearchListVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Backend.Data.Models;

namespace Backend.Data.ViewModels.Response
{
    /// <summary>    
    /// View Model class for the list of users that results from a search
    /// </summary>
    public class UserSearchListVM
    {
        public List<UserResultVM> Users { get; set; }


        /// <summary>
        /// Class' constructor method
        /// </summary>
        public UserSearchListVM()
        {
            this.Users = new List<UserResultVM>();
        }
    }
}